### PR TITLE
refactor:Redux state management provided

### PR DESCRIPTION
**Pull request by dev-enforced**
In this Pull request reviewer would be able to get an overview of the files related to implementation of the Redux state management system and simultaneous removal of the Context+Reducer State management system. 
Reviewer can see the live preview links to check these simple feature updates:
1) Theme Change
2) Sidebar view on mobile phones
3) Videos and Categories fetch up

Links to see: https://mi-stream-git-redux-refactor-dev-enforced.vercel.app/
https://mi-stream-git-redux-refactor-dev-enforced.vercel.app/explore

### CODE REVIEW

File: src/reduxFiles/reducers/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export { updateTheme } from "./theme/themeSlice";
+export { updateSidebarView } from "./sidebar/sidebarSlice";
+export { getCategories } from "./categories/categoriesSlice";
+export { getVideos } from "./videos/videosSlice";
+// @supabase/supabase-js
+// Create a project
```
Comment (sushil-mod) [inline src/reduxFiles/reducers/index.js]: you remove this comment if it's not for future refrence

File: src/reduxFiles/reducers/videos/videosSlice.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      data: { videos: videosAvailable },
+    } = await receiveAllVideos();
+    return videosAvailable;
+  } catch (videoSetupError) {
+    console.error("ERROR OCCURED WHILE SETTING UP VIDEOS:", videoSetupError);
+    return Promise.reject(videoSetupError);
+  }
+};
+
+const getVideos = createAsyncThunk("videos/getVideos", getVideosService);
```
Comment (sushil-mod) [inline src/reduxFiles/reducers/videos/videosSlice.js:19]: nicely used createAsyncThunk

**Review by sushil-mod — APPROVED** [review]
overall good refactoring of code from context to redux. Good job

**Review by muskaanshah — APPROVED** [review]
Good job on converting context to redux
Design suggestion - categories on home page aren't clickable and the length of the boxes vary

**Comment by dev-enforced** [comment]
> Good job on converting context to redux Design suggestion - categories on home page aren't clickable and the length of the boxes vary

Yes @muskaanshah category linking feature hasn't been added yet!! The design suggestion I will try to implement in the next PR related to category linking action.

**Pull request merged**